deploy/iroha/genesis.block

```diff
@@ -137,30 +137,36 @@
                 "createRole": {
                   "roleName": "tester",
                   "permissions": [
-                    "can_get_all_accounts",
-                    "can_get_all_acc_ast",
-                    "can_add_asset_qty",
-                    "can_transfer",
-                    "can_create_asset",
                     "can_create_account",
+                    "can_set_detail",
+                    "can_create_asset",
+                    "can_transfer",
                     "can_receive",
-                    "can_read_assets",
-                    "can_append_role",
+                    "can_add_asset_qty",
+                    "can_subtract_asset_qty",
+                    "can_create_domain",
+                    "can_grant_can_add_my_signatory",
+                    "can_grant_can_remove_my_signatory",
+                    "can_grant_can_set_my_quorum",
+                    "can_grant_can_transfer_my_assets",
                     "can_add_peer",
-                    "can_set_detail",
+                    "can_append_role",
+                    "can_create_role",
+                    "can_detach_role",
+                    "can_add_signatory",
+                    "can_remove_signatory",
+                    "can_set_quorum",
+                    "can_get_all_acc_detail",
+                    "can_get_all_accounts",
+                    "can_get_all_acc_ast",
+                    "can_read_assets",
                     "can_get_blocks",
+                    "can_get_roles",
+                    "can_get_all_signatories",
                     "can_get_domain_accounts",
-                    "can_set_quorum",
-                    "can_add_signatory",
-                    "can_create_account",
                     "can_get_all_txs",
                     "can_get_domain_acc_detail",
-                    "can_read_assets",
-                    "can_grant_can_set_my_quorum",
-                    "can_grant_can_add_my_signatory",
-                    "can_grant_can_transfer_my_assets",
-                    "can_get_my_signatories",
```

Comment:
it's deleted and not added anywhere else

Reply:
there is can_get_all_signatories